src/components/fed/__test__/fed.test.js

```diff
@@ -0,0 +1,141 @@
+import React from "react";
+import { act, fireEvent } from "@testing-library/react";
+import { Fed } from "../";
+import { renderWithProvider, setWindowWidth } from "../../../utils/testing";
+import { getToday, getHour, getUserName } from "../../../utils/fed";
+
+jest.useFakeTimers();
+
+describe("Fed", () => {
+  let wrapper;
+  beforeEach(() => {
+    setWindowWidth(992); //md
+    wrapper = renderWithProvider(<Fed />);
+  });
+
+  it("should show Fed", () => {
+    expect(wrapper).toBeTruthy();
+  });
+
+  describe("show titles and subTitles", () => {
```

Comment:
este describe mejor cambiarlo por un solo it llamado algo asi como "renders ComponentName elements"
y ponest todos los expect en el mismo it